TELCODOCS-959: Openshift on AWS Outpost support (#55427)
openshift/openshift-docs: modules/installation-aws-outposts-supported-instance-types.adoc

> // Module included in the following assemblies:

Reviewer: Suggest updating the file name to `installation-identify-supported-instance-types`

Author: File renamed as suggested

//
// installing/installing_aws/installing-aws-outposts-remote-workers.adoc

> [id="installation-aws-outposts-supported-instance-types_{context}"]

Reviewer: ```suggestion
:_content-type: PROCEDURE
[id="installation-identify-supported-instance-types_{context}"]
```

Author: :_content-type: PROCEDURE added

= Identifying your AWS Outposts instance types

AWS Outposts rack catalog includes options supporting the latest generation Intel powered EC2 instance types with or without local instance storage.

> You need to validate first which instance types are configured in your AWS Outpost instance.

Reviewer: ```suggestion
Identify which instance types are configured in your AWS Outpost instance. As part of the installation process, you must update the `install-config.yaml` file with the instance type that the installation program will use to deploy worker nodes.
```

Author: Line 9 updated

> Use the AWS CLI to get the list of supported instance types

Reviewer: ```suggestion
.Procedure
Use the AWS CLI to get the list of supported instance types by running the following command:
```

Author: . Procedure added

[source,terminal]
----

> aws outposts get-outpost-instance-types --outpost-id <outpost_id> <1>

Reviewer: ```suggestion
$ aws outposts get-outpost-instance-types --outpost-id <outpost_id> <1>
```

Author: $ added
